.gitignore

```diff
@@ -1,3 +1,18 @@
 .env
```

Comment:
Can you help me to put these three lines into some section accordingly please?

Reply:
it just a template that jetbrain suggest lul

Reply:
Please revise

Reply:
-__-